ts/screens/wallet/FirstTransactionSummaryScreen.tsx

```diff
@@ -0,0 +1,261 @@
+          tranche={this.state.tranche}
+        />
+      );
+    } else {
+      return (
+        <PaymentSummaryComponent
+          navigation={this.props.navigation}
+          amount={this.state.amount}
+          expireDate={this.state.expireDate}
+          tranche={this.state.tranche}
+        />
+      );
+    }
+  }
+
+  public render(): React.ReactNode {
+    // The following data should be globally obtained as result of the transaction identification.
+    // Further discussions on thier availability is required
+    const NOMEAVVISO: string = "Tari 2018";
+    const NOMEENTE: string = "Comune di Gallarate - Settore Tributi";
+    const INDIRIZZOENTE: string = "Via Cavour n.2 - Palazzo Broletto,21013";
+    const CITTAENTE: string = "Gallarate (VA)";
+    const TELENTE: string = "0331.754224";
+    const WEBPAGEENTE: string = "www.comune.gallarate.va.it";
+    const EMAILENTE: string = "tributi@coumne.gallarate.va.it";
+    const PECENTE: string = "protocollo@pec.comune.gallarate.va.it";
+    const DESTINATARIO: string = "Mario Rossi";
+    const INDIRIZZODESTINATARIO: string = "Via Murillo 8, 20149 Milano (MI)";
+    const CBILL: string = "A0EDT";
+    const IUV: string = "111116000001580";
+    const CODICEENTE: string = "01199250158";
+
+    return (
+      <Container>
+        <AppHeader>
+          <Left>
+            <Button transparent={true} onPress={() => this.goBack()}>
+              <Icon name="chevron-left" />
+            </Button>
+          </Left>
+          <Body>
+            <Text>{I18n.t("wallet.firstTransactionSummary.header")}</Text>
+          </Body>
+        </AppHeader>
+
+        <Content original={true}>
+          <Grid style={styles.padded}>
+            <Row>
+              <Left>
+                <H3>{I18n.t("wallet.firstTransactionSummary.title")}</H3>
+                <H1>{NOMEAVVISO}</H1>
+              </Left>
+              <Right>
+                <Image
+                  source={require("../../../img/wallet/icon-avviso-pagopa.png")}
+                />
+              </Right>
+            </Row>
+            <View spacer={true} large={true} />
+          </Grid>
+          {this.getSummary()}
+          <View spacer={true} large={true} />
+          <Grid style={styles.padded}>
+            <Row>
+              <Text bold={true}>
+                {I18n.t("wallet.firstTransactionSummary.entity")}
+              </Text>
+            </Row>
+            <Row>
+              <Text>{NOMEENTE}</Text>
+            </Row>
+            <Row>
+              <Text>{INDIRIZZOENTE}</Text>
+            </Row>
+            <Row>
+              <Text>{CITTAENTE}</Text>
+            </Row>
+            <Row>
+              <Text>{I18n.t("wallet.firstTransactionSummary.info")}</Text>
+            </Row>
+            <Row>
+              <Text>{I18n.t("wallet.firstTransactionSummary.tel")}</Text>
+              <Text> {""} </Text>
+              <Text link={true}>{TELENTE}</Text>
```

Comment:
I am not sure but i think text can be nested like:

<Text>{I18n.t("wallet.firstTransactionSummary.tel")} <Text link={true}>{TELENTE}</Text></Text>

Reply:
https://facebook.github.io/react-native/docs/text.html#nested-text